packages/ds-ext/src/DiagramEditorProvider.ts

```diff
@@ -75,9 +77,6 @@ export class DiagramEditorProvider implements vscode.CustomEditorProvider<Diagra
       vscode.Uri.file(mainScript)
```

Comment:
This replaces window.initalData? Neat 👍

Reply:
No, I realized that `fileUri` isn't necessary 👀 
```ts
window.initialData = {
     fileUri: "${fileUri}",  // Pass file URI
};
```